Sources/TensorFlow/Core/TensorGroup.swift

```diff
@@ -35,7 +35,7 @@ public protocol TensorArrayProtocol {
     var _tensorHandles: [_AnyTensorHandle] { get }
 
     init(_owning tensorHandles: UnsafePointer<CTensorHandle>?, count: Int)
-    init<C: RandomAccessCollection>(_handles: C) where C.Element == _AnyTensorHandle
+    init<C: RandomAccessCollection>(_handles: C) where C.Element : _AnyTensorHandle
```

Comment:
The Google Swift Style Guide recommends no whitespaces before colons in a conformance constraint.

```suggestion
    init<C: RandomAccessCollection>(_handles: C) where C.Element: _AnyTensorHandle
```